CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [13.1.0](https://github.com/commercetools/merchant-center-application-kit/compare/v13.0.2...v13.1.1) (2019-04-09)
+
+#### 💅 Type: Enhancement
```

Comment:
sorry `Type: Fix` I guess. Depending on what we pick.

Reply:
I personally think it's a feature 😬

Reply:
I see it more as a feature rather than a fix TBH 😅

Reply:
If feature then it is 13.2. The confusion comes from the PR having false tags then.

Reply:
done ccdfb96

Reply:
oops. did a push without `git add` 😆